extensions/ui/org.eclipse.smarthome.ui.basic/ESH-INF/i18n/basic.properties

```diff
@@ -1,4 +1,5 @@
 main.offline-msg = Offline: waiting for connection to become available
+main.sse-limit-msg = Server limit reached: failback to long polling mode
```

Comment:
maybe "fallback"?

Reply:
Ok.
Maybe this current message is too much technical ?
What do you think about "Run in not nominal mode due to server limit reached" ?

Reply:
I would rather say "SSE limit on server reached, running in fallback mode"

This way the ordinary user knows that "a limit" has been reached and that the system is still working but in a non desired "fallback" mode. It still preserves the technical information about the SSE limit, though.

Reply:
I finally did not mention the limit because I want to cover any error that could occur when subscribing a SSE (even if currently the SSE limit should be the only that could happen).

But I like the second part of your text that does not mention "long polling".

What do you think about "SSE subscription failed, running in fallback mode" ?

Reply:
I am fine with it.